about.html

```diff
@@ -0,0 +1,124 @@
+                    <a href="./index.html#partners" class="menu-icons">Partners</a></li>
+                <li><img src="./src/hamburger-logos/about.png" alt="about-btn">
+                    <a href="./about.html" class="menu-icons">About</a></li>
+            </ul>
+        </div>
+    </div>
+
+        <main>
+            <div class="first-container">
+            <section id="about">
+                <span class="greetings">"Hello world !"</span>
+                <h1>WEBuild</h1>
+                <span class="seperator1"></span>
+                <p class="website-main-descrption">
+                    We’re an online school that teaches web development. 
+                    If you graduate from us we would provide you with verified 9 certificates
+                    that come in handy when you are about to get the job you’ve always wanted 
+                    as a web development engineer. 
+                    <span class ="additional-for-desktop-only">However you do not only teach you that.
+                    We would teach you other various soft skills that are as important as the coding skills
+                    you would learn from our extended curriculum. The latter is composed of 9 Modules that
+                    will once finished will grant you mastery in : HTML, JS, CSS, Ruby, RoR, React, Redux.. 
+                    and more different famous frameworks that you might need in your future job. We started big
+                    and we’re only bigger now. We helped thousands of pupils become full stack web developers 
+                    and land their dream job in a few months. PS: This project is a Bootcamp template website.
+                    If by any chance you happened to land on this page be 
+                    aware that everything written above is part of a project and does not relate to actual events.
+                </span>
+                </p>
+                <p class="contact-info">Please contact us on Email if you need help</p>
+                <div class="mail">
+                <a href="mailto:Amino.bhl@gmail.com">Amino.bhl@gmail.com </a>
+                <span>></span>
+                 </div>
+            </section>
+               </div>
+                <div class="second-container">
+                 <section id="logo">
+                 <h2 class="logo-info">WEBuild Logo</h2>
+                <span class="seperator2"></span>
+                <p class="logo-description">
+                    The logo is manually created by hand and photo-edited using Photopea (online photo
+                     editing website). the mini computer logo was generated using Adobe sparks Logo Creator
+                     online under free non-commercial use lisence.
+                </p>
+                <a href="./index.html"><img class="logo1" src="./src/logo/main-logo.svg" alt="main-logo"></a>
+            </section>
+        </div>
```

Comment:
- [ ] OPTIONAL: You have missed the See past events section on the about page. To make your project complete I strongly suggest you add it. :pray: 
![image](https://user-images.githubusercontent.com/64894927/132577414-da871cce-f94c-4065-8903-96b8dc7f731b.png)
![image](https://user-images.githubusercontent.com/64894927/132577490-ddca43a5-75d9-458b-858f-5e435b3a4d01.png)

Reply:
Hey @henatan99, I did not forget the section in the about me I ommited it. The website is a bootcamp there are no past events to mention in there.
Now I have had 3 code reviews one of which literally asked me to change 2 out of 3 stuff that I have already changed prior to his comment and you are literally stoping the project for 10px margin which I have added on purpose just to make the design look cooler and more matching becasue the original one has an "@" in stead of that space knowing that small changes are allowed

Reply:
I am so sorry I didn't notice that it was your last review. But, the missing part was just an optional requirement though it was important. You could have populated it with something that matches your website because it is a design requirement and you should stick with it. And you also didn't make one change requested by previous reviewer. I am sorry I cannot change my review, but you can write second opinion to student success or code review leaders and they may take into consideration and change your project status if they are convinced by your second opinion. Can you do that?

Reply:
@hentan99 it's okey it's a capstone project so I have 5 attempts. The code reviewer said : 
"The typography in your hero/ top section does not match the provided desgn template. We actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text, as can be seen in the provided template design. Let's ensure that the font sizes, font weights, font colors, and letter spacing matches that of the provided template design"
where he could just say add the first line and fix margins..
Next did not see that and then next so i thought the problem is fixed but then it wasn't..Next said change stuff I've already changed. Now with that said I thank you all for your cooperation and time. In anyways I fixed the issue with the margin that you have requested for and I sent another code review attempt